### PR TITLE
[IMP] mail: `Message`, remove note style

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Prior to this commit, the note had its own style. This created too much
confusion with the other message types.

This commit removes the note style to improve the readability in the
chatter.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hey @alexkuhn 👋 
Could you take a look at it ? :)

File: addons/mail/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js [hunk 1]
```diff
@@ -928,7 +928,7 @@ QUnit.test('chat window: scroll conservation on toggle discuss', async function
 
     const pyEnv = await startServer();
     const mailChannelId1 = pyEnv['mail.channel'].create({});
-    for (let i = 0; i < 10; i++) {
+    for (let i = 0; i < 100; i++) {
```
Comment (alexkuhn) [inline addons/mail/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js:931]: Test hard-checked scroll happens with precise scroll position.
It assumes there are enough messages to have at least a specific scroll height.

Before this PR, 10 messages were enough for `142`, but now it's too low (just does about `129`).

So I just changed it to `100`, so it's definitely enough for making a scroll change to `142`.

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo retry

**Comment by alexkuhn** [comment]
> @Brieuc-brd @alexkuhn staging failed: ci/runbot (view more at https://runbot.odoo.com/runbot/build/18675759)

🤦‍♂️

**Comment by alexkuhn** [comment]
@robodoo retry

**Pull request closed without merging**